lib/git.js

```diff
@@ -4,9 +4,12 @@ const fs = Promise.promisifyAll(require('fs'))
 const tmp = Promise.promisifyAll(require('temp').track())
 const gh = require('github-url-to-object')
 
-const NOT_A_GIT_REPOSITORY = 'Not a git repository'
+const NOT_A_GIT_REPOSITORY = 'not a git repository'
```

Comment:
In newer versions of Git, the error message is `fatal: not a git repository [...]`.

Thus, I am removing the capital `N` and comparing this string against the lowercased version of the error message (see below).